.github/release-changelog.sh

```diff
@@ -6,4 +6,4 @@ cd "$(dirname "${BASH_SOURCE[0]}")/.."
 set -x
 
 _chart_version=$(yq eval .version "charts/vector/Chart.yaml")
-git cliff --config .github/cliff.toml -t "vector-$_chart_version" -p CHANGELOG.md -u
+git cliff --config .github/cliff.toml --tag "vector-$_chart_version" --prepend CHANGELOG.md --unreleased
```

Comment:
👍 thanks for making these the long form.

Reply:
I mean... since I had to check the help text to see what I was doing it seemed a good choice 😄